tests/active_features/owner.feature

```diff
@@ -20,6 +20,7 @@ Feature: Set a site's owner
   Scenario: Failing to change the site owner when there is only one team member
     Given "[[other_user]]" is not a member of the team on "[[test_site_name]]"
     When I run "terminus owner:set [[username]] --site=[[test_site_name]]"
+    When This step is implemented I will test: terminus owner:set
     Then I should get:
     """
     The new owner must be added with "terminus site team add-member" before promoting.
```

Comment:
As we are no longer making Terminus do this checking, we can remove this test and the tests/fixtures/site_set-owner_solo fixture file.